### DISCUSSION TITLE
A list to put the people who using government funds in US/Canada

### DISCUSSION THREAD

**Discussion by mostafasoufi** [WeAreMahsaAmini/Core#281] [Ideas]
The title is quite cristal clear but I'll add some things

1. Name of the people and the country where they live
2. A screenshot or a photo of the document as proof
3. Create a dynamic text for reporting
4. Short URL for sharing in social network
5. Search people

Any idea? let's talk!

**Comment by AnonymousMaximus2** [comment]
HaHa, 

you think it is easy to find that?

if you can find any person uses government funds in any other country,

you can report to western government and if they can block their fund, you can earn at least $USD 30,000 per each person that you are reporting.

for large amount of funds the commission and incentives is huge.

not only they are very intelligent but also if someone finds this information they can report and get commission for themselves.

**Comment by PGorj** [comment]
Honestly, we need to report the sons/daughters, grandchildren of these mulahs to US/Europe/Canadian authorities who live, study, and work in their countries with stolen/blood money. Many people have already flagged a few of the grand daughters of these murderers... how can we not get some substantial dirt on them? what is the best way to flag these to authorities? we need lawyers to help us!

**Reply by mostafasoufi** [reply]
Yes, I meant **sons/daughters, grandchildren**

**Comment by AnonymousMaximus2** [comment]
If you know the facts behind Iranian economy you can easily forgive their sons and daughters, 

all of their sons and daughters accumulated money may be maximum Billions of $USD

Thousands of Milllion $USD equals just $USD 1 Billion,

but it is nothing,...

But Iran has Trillions of $USD money that during decades by selling Oil to international markets earned, 

This huge money circulating around the world in hands of Russia, China, and some time small amounts in India, South Korea, Turkey, ...

Think about that, 

Think BIG


I suggest count Iranian Government assets abroad to estimate their economic power,


Remember Sun Tzu China Army General and Commander said 2500 years ago : 

If you know the enemy and know yourself, you need not fear the result of a hundred battles.
